### PR TITLE
Add `react-popper` wrappers

### PR DESCRIPTION
**Pull request by turansky**
cc @sgrishchenko @aerialist7

### CODE REVIEW

File: kotlin-react-popper/src/main/kotlin/popper/PopperOptions.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package popper
+
+import kotlinext.js.ReadonlyArray
+
+external interface PopperOptions<Modifiers> {
+    var placement: Placement
+    var modifiers: ReadonlyArray<Modifier<Modifiers>> // $Shape
+    var strategy: PositioningStrategy
+    var onFirstUpdate: (State) -> Unit // $Shape
```
Comment (sgrishchenko) [inline kotlin-react-popper/src/main/kotlin/popper/PopperOptions.kt:9]: type alias `PopperState`?

Comment (turansky) [reply kotlin-react-popper/src/main/kotlin/popper/PopperOptions.kt:9]: It's `State` from `popper` package

Comment (turansky) [reply kotlin-react-popper/src/main/kotlin/popper/PopperOptions.kt:9]: `PopperState` looks better

**Comment by Leonya** [comment]
orgjetbrainskotlin-2002

**Pull request merged**